### PR TITLE
add missing back button in ManageAccountsActivity & UserInfoActivity

### LINKED ISSUE #5774: Manage Accounts & User Info are missing back button
Thanks for finding @AndyScherzinger

### PR DESCRIPTION
**Pull request by chawki505**
Fix #5774 

I added the missing buttons in Manage Accounts Activity & UserInfo Activity.

If there is something I can do don't hesitate to tell me in this PR.

### Testing 

- [X] Tests not needed

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java [hunk 1]
```diff
@@ -123,8 +126,23 @@ protected void onCreate(Bundle savedInstanceState) {
         recyclerView = findViewById(R.id.account_list);
 
         setupToolbar();
+
+        // -- Action bar setup --
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java:130]: Those comments are superficial.

File: src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java [hunk 2]
```diff
@@ -421,11 +439,11 @@ private void performAccountRemoval(User user) {
         bundle.putString(AccountRemovalJob.ACCOUNT, user.getAccountName());
 
         new JobRequest.Builder(AccountRemovalJob.TAG)
-                .startNow()
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java:424]: Whitespace change will cause conflicts in with other PRs touching job scheduler.

File: src/main/java/com/owncloud/android/ui/activity/UserInfoActivity.java [hunk 3]
```diff
@@ -145,10 +148,25 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.user_info_layout);
         unbinder = ButterKnife.bind(this);
 
+
+        // Action bar setup
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/ui/activity/UserInfoActivity.java:152]: superficial comments

File: src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java (hunk 1 above)
Comment (chawki505) [reply src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java:130]: i will remove this

File: src/main/java/com/owncloud/android/ui/activity/UserInfoActivity.java (hunk 3 above)
Comment (chawki505) [reply src/main/java/com/owncloud/android/ui/activity/UserInfoActivity.java:152]: i will remove this

File: src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java (hunk 2 above)
Comment (chawki505) [reply src/main/java/com/owncloud/android/ui/activity/ManageAccountsActivity.java:424]: Ah sorry, I will fix that

**Comment by chawki505** [comment]
@ezaquarii @tobiasKaminsky @AndyScherzinger it's OK for you ?

**Comment by AndyScherzinger** [comment]
Code reviews are in and tested successfully.

@chawki505 Welcome to Nextcloud and thanks a lot for the contribution! This is highly appreciated ❤️

**Pull request merged**